docs/manual/external-tools.tex

```diff
@@ -391,6 +394,18 @@ \section{IntelliJ IDEA\label{intellij}}
 \url{https://youtrack.jetbrains.com/issue/IDEA-159286}.
 
 
+\subsection{Troubleshooting IntelliJ\label{intellij-troubleshooting}}
+
+\begin{enumerate}
+\item
+  If certain expected warnings are ignored when build with IntelliJ( such as str.charAt(i)
```

Comment:
This isn't needed because a warning will now be issued.  (This is a recent change, see https://github.com/typetools/checker-framework/pull/1888)